### PR TITLE
Cleanup2

### PR DESCRIPTION
**Pull request by DougInAMug**
- unify src subfolder names and update imports
- remove unused files
- update .gitignore
- update index.html

### CODE REVIEW

File: src/components/PaginationMovies.tsx
Comment (verascheunert) [inline src/components/PaginationMovies.tsx:1]: I know it's not related to your changes, but could you move the key (l.43) one line up to the Link please?

**Review by verascheunert — COMMENTED** [review]
Only one small comment unrelated to your changes that you could change other than that, lgtm!

File: src/components/PaginationMovies.tsx
Comment (DougInAMug) [reply src/components/PaginationMovies.tsx:1]: Should be done now!

However I feel kind of bad to have to have so many other PR's being affected :(

**Comment by verascheunert** [comment]
Lgtm :)

**Review by verascheunert — APPROVED** [review]

**Pull request merged**